specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-05-04/examples/CosmosDBDatabaseAccountCreateMax.json

```diff
@@ -0,0 +1,133 @@
+{
+  "parameters": {
+    "accountName": "ddb1",
+    "resourceGroupName": "rg1",
+    "api-version": "2020-05-04",
+    "subscriptionId": "subid",
+    "createUpdateParameters": {
+      "location": "westus",
+      "tags": {},
+      "kind": "GlobalDocumentDB",
+      "properties": {
+        "databaseAccountOfferType": "Standard",
+        "ipRules": [
```

Comment:
should be "ipRules": [
 { "ipAddressOrRange": "10.10.10.10" },
{ "ipAddressOrRange": "12.24.48.0/10" }
]